File: W2/request3.js
function maxProduct(nums){
    var each = 0;
    var sum = [];
    for(i=0; i<nums.length; i++){
        for(j=i+1; j<nums.length; j++){
            each=nums[i]*nums[j];
            sum.push(each)
        }
    }
    console.log(Math.max.apply(null, sum))
}
maxProduct([5, 20, 2, 6]);
maxProduct([10, -20, 0, 3]);

File: W2/Request3.py
def main():
    def maxProduct(nums):
        max = 0
        num = 0
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                num = nums[i]*nums[j]
                if i==0 and j==1:
                    max = num
                if num > max:
                    max = num
        print(max)


if __name__ == '__main__':
    main()